launchpad/errors.gno

```diff
@@ -0,0 +1,26 @@
+package launchpad
+
+import (
+	"errors"
+
+	"gno.land/p/demo/ufmt"
+)
+
+var (
+	errNoPermission        = errors.New("[GNOSWAP-LAUNCHPAD-001] no permission")
+	errNotRegistered       = errors.New("[GNOSWAP-LAUNCHPAD-002] not registered")
+	errAlreadyRegistered   = errors.New("[GNOSWAP-LAUNCHPAD-003] already registered")
+	errDataNotFound        = errors.New("[GNOSWAP-LAUNCHPAD-004] data not found")
+	errInactiveProject     = errors.New("[GNOSWAP-LAUNCHPAD-005] inactive project")
+	errInactiveTier        = errors.New("[GNOSWAP-LAUNCHPAD-006] inactive tier")
+	errInvalidInput        = errors.New("[GNOSWAP-LAUNCHPAD-007] invalid input")
+	errDuplicateProject    = errors.New("[GNOSWAP-LAUNCHPAD-008] duplicate project")
```

Comment:
Do we block this? I think it's duplicatable via contract

Reply:
to identify project, we use `projectId` which is form of `tokenPath_createHeight`.
therefore it can not be duplicated as view from contract.